<!DOCTYPE html>
<html lang='en' ><meta charset="utf-8">
<meta name="viewport" content="width=device-width">


<title>Genetic Factors that Confer PAH Metabolism in Thalassospira tepidiphila | Erica Littman</title>
<link rel="stylesheet" href="https://ericalittman.com/css/eureka.min.css">
<script defer src="https://ericalittman.com/js/eureka.min.js"></script>

<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link rel="preload"
  href="https://fonts.googleapis.com/css2?family=Lora:wght@400;600;700&family=Noto+Serif+SC:wght@400;600;700&display=swap"
  as="style" onload="this.onload=null;this.rel='stylesheet'">

<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.1.0/build/styles/solarized-light.min.css%25!%28EXTRA%20string=solarized-light%29"
   media="print"
  onload="this.media='all';this.onload=null" crossorigin>
<script defer src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.1.0/build/highlight.min.js"
   crossorigin></script>

  <script defer src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.1.0/build/languages/dart.min.js"
     crossorigin></script>

<script defer src="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.14.0/js/all.min.js"
   integrity="sha256-uNYoXefWRqv&#43;PsIF/OflNmwtKM4lStn9yrz2gVl6ymo="  crossorigin></script>


<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css"
   integrity="sha384-AfEj0r4/OFrOo5t7NnNe46zW/tFgW6x/bCJG8FqQCEo3&#43;Aro6EYUG4&#43;cU&#43;KJWu/X"  media="print"
  onload="this.media='all';this.onload=null" crossorigin>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.js" 
  integrity="sha384-g7c&#43;Jr9ZivxKLnZTDUhnkOnsh30B4H0rpLUpJ4jAIKs4fnJI&#43;sEnkvrMWph2EDg4"  crossorigin></script>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/contrib/auto-render.min.js"
   integrity="sha384-mll67QQFJfxn0IYznZYonOWZ644AWYC&#43;Pt2cHqMaRhXVrursRwvLnLaebdGIlYNa"  crossorigin></script>
<script>
  document.addEventListener("DOMContentLoaded", function () {
    renderMathInElement(document.body, {
      delimiters: [
        { left: "$$", right: "$$", display: true },
        { left: "$", right: "$", display: false },
        { left: "\\(", right: "\\)", display: false },
        { left: "\\[", right: "\\]", display: true }
      ],
    });
  });
</script>


<link rel="icon" type="image/png" sizes="32x32" href="https://ericalittman.com/images/icon_hu933212be7dc26f7a295562446aa78b7b_1602_32x32_fill_box_center_2.png">
<link rel="apple-touch-icon" sizes="180x180" href="https://ericalittman.com/images/icon_hu933212be7dc26f7a295562446aa78b7b_1602_180x180_fill_box_center_2.png">

<meta name="description"
  content="Post with featured image.">
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [{
      "@type": "ListItem",
      "position": 1 ,
      "name":"Posts",
      "item":"https://ericalittman.com/posts/"},{
      "@type": "ListItem",
      "position": 2 ,
      "name":"Genetic Factors that Confer PAH Metabolism in Thalassospira tepidiphila",
      "item":"https://ericalittman.com/posts/proposal/"}]
}
</script>



<script type="application/ld+json">
{
    "@context": "https://schema.org",
    "@type": "Article",
    "mainEntityOfPage": {
        "@type": "WebPage",
        "@id": "https://ericalittman.com/posts/proposal/"
    },
    "headline": "Genetic Factors that Confer PAH Metabolism in Thalassospira tepidiphila | Erica Littman",
    "image": "https://ericalittman.com/images/phytoplankton_2020228_web.jpg",
    "datePublished": "2018-03-05T00:00:00+00:00",
    "dateModified": "2018-03-05T00:00:00+00:00",
    "wordCount":  1705 ,
    "publisher": {
        "@type": "Person",
        "name": "E. Littman",
        "logo": {
            "@type": "ImageObject",
            "url": "https://ericalittman.com/images/icon.png"
        }
        },
    "description": "Post with featured image."
}
</script><meta property="og:title" content="Genetic Factors that Confer PAH Metabolism in Thalassospira tepidiphila | Erica Littman" />
<meta property="og:type" content="article" />


<meta property="og:image" content="https://ericalittman.com/images/icon.png">


<meta property="og:url" content="https://ericalittman.com/posts/proposal/" />



<meta property="og:description" content="Post with featured image." />



<meta property="og:locale" content="en" />




<meta property="og:site_name" content="Erica Littman" />






<meta property="article:published_time" content="2018-03-05T00:00:00&#43;00:00" />


<meta property="article:modified_time" content="2018-03-05T00:00:00&#43;00:00" />



<meta property="article:section" content="posts" />


<meta property="article:tag" content="marketing" />

<meta property="article:tag" content="social media" />

<meta property="article:tag" content="portfolio" />





<body class="flex flex-col min-h-screen">
  <header class="fixed flex items-center w-full min-h-16 pl-scrollbar z-50 bg-secondary-bg shadow-sm">
    <div class="w-full max-w-screen-xl mx-auto"><script>
    let storageColorScheme = localStorage.getItem("lightDarkMode")
    if ((storageColorScheme == 'Auto' && window.matchMedia("(prefers-color-scheme: dark)").matches) || storageColorScheme == "Dark") {
        document.getElementsByTagName('html')[0].classList.add('dark')
    }
</script>
<nav class="flex items-center justify-between flex-wrap px-4 py-4 md:py-0">
    <a href="/" class="mr-6 text-primary-text text-xl font-bold">Erica Littman</a>
    <button id="navbar-btn" class="md:hidden flex items-center px-3 py-2" aria-label="Open Navbar">
        <i class="fas fa-bars"></i>
    </button>

    <div id="target"
        class="hidden block md:flex md:flex-grow md:justify-between md:items-center w-full md:w-auto text-primary-text z-20">
        <div class="md:flex md:h-16 text-sm md:flex-grow pb-4 md:pb-0 border-b md:border-b-0">
            <a href="/#about" class="block mt-4 md:inline-block md:mt-0 md:h-(16-4px) md:leading-(16-4px) box-border md:border-t-2 md:border-b-2  border-transparent  mr-4">About</a>
            <a href="/posts/" class="block mt-4 md:inline-block md:mt-0 md:h-(16-4px) md:leading-(16-4px) box-border md:border-t-2 md:border-b-2  selected-menu-item  mr-4">Posts</a>
        </div>

        <div class="flex">
            <div class="relative pt-4 md:pt-0">
                <div class="cursor-pointer hover:text-eureka" id="lightDarkMode">
                    <i class="fas fa-sun"></i>
                </div>
                <div class="fixed hidden inset-0 opacity-0 h-full w-full cursor-default z-30" id="is-open">
                </div>
                <div class="absolute flex flex-col left-0 md:left-auto right-auto md:right-0 hidden bg-secondary-bg w-48 rounded py-2 border border-tertiary-bg cursor-pointer z-40"
                    id='lightDarkOptions'>
                    <span class="px-4 py-1 hover:text-eureka">Light</span>
                    <span class="px-4 py-1 hover:text-eureka">Dark</span>
                    <span class="px-4 py-1 hover:text-eureka">Auto</span>
                </div>
            </div>
        </div>
    </div>

    <div class="fixed hidden inset-0 opacity-0 h-full w-full cursor-default z-0" id="is-open-mobile">
    </div>

</nav>
<script>
    let element = document.getElementById('lightDarkMode')
    if (storageColorScheme == 'Auto') {
        element.firstElementChild.classList.remove('fa-sun')
        element.firstElementChild.setAttribute("data-icon", 'adjust')
        element.firstElementChild.classList.add('fa-adjust')
        document.addEventListener('DOMContentLoaded', () => {
            switchMode('Auto')
        })
    } else if (storageColorScheme == "Dark") {
        element.firstElementChild.classList.remove('fa-sun')
        element.firstElementChild.setAttribute("data-icon", 'moon')
        element.firstElementChild.classList.add('fa-moon')
    }
    
    document.addEventListener('DOMContentLoaded', () => {
        getcolorscheme();
        switchBurger();
    });
</script></div>
  </header>
  <main class="flex-grow pt-16">
    <div class="pl-scrollbar">
      <div class="w-full max-w-screen-xl lg:px-4 xl:px-8 mx-auto">


<div class="grid grid-cols-2 lg:grid-cols-8 gap-4 lg:pt-12">
    <div
        class="col-span-2  lg:col-span-6 bg-secondary-bg rounded px-6 py-8">
        <h1 class="font-bold text-3xl text-primary-text">Genetic Factors that Confer PAH Metabolism in Thalassospira tepidiphila</h1>
        <div class="flex flex-wrap flex-row items-center mt-2 text-tertiary-text">
    <div class="mr-6 my-2">
        <i class="fas fa-calendar mr-1"></i>
        <span>2018-03-05</span>
    </div>
    <div class="mr-6 my-2">
        <i class="fas fa-clock mr-1"></i>
        <span>9 min read</span>
    </div>
    
    

    
</div>
        
        

        <div class="content">
            <h2 id="background">Background</h2>
<p>This is a grant proposal I wrote for the course Advanced Molecular Microbiology at Northwestern University on using phytoplankton as a microplastic removal tactic in marine ecosystems.</p>
<h2 id="genetic-factors-that-confer-pah-metabolism-in-thalassospira-tepidiphila">Genetic Factors that Confer PAH Metabolism in Thalassospira tepidiphila</h2>
<p>Phytoplankton make up a large portion of the biomass of marine primary producers; therefore, maintaining a healthy population is essential for the stability of the food chain1. Polycyclic aromatic hydrocarbons (PAHs) enter marine ecosystems via oil spills and other pollutants such as the burning of plastics. Increasing PAH concentrations have a negative effect on phytoplankton populations and the overall community structure due to the organism’s lipophilic attributes1,8. Such documented negative effects include a delay or complete interruption of DNA synthesis and cellular replication, and, in turn, increased cell mortality and a resultant decrease of population size15. A marine bacterium first classified in 2008, Thalassospira tepidiphila, was shown to have PAH degrading abilities in petroleum-contaminated seawater2. Currently, the genetic factors contributing to the strain’s ability to degrade PAHs remain unknown. In this project, I will identify and characterize the genetic factors involved in PAH metabolism via genetic knockouts and sequencing of colonies with a decreased ability to metabolize such hydrocarbons. Additionally, I will characterize the effects of the presence of T. tepidiphila on two strains of marine phytoplankton—Prochlorococcus marinus and Micromonas pusilla—in a PAH-polluted marine microcosm.</p>
<h3 id="objectives">Objectives</h3>
<p>I will cultivate and confirm PAH metabolic function in T. tepidiphila to identify and quantify metabolites that are produced by degradation of three common PAHs found in marine ecosystems: naphthalene, fluorene, phenanthrene, and dibenzothiophene. I will characterize genes conferring PAH metabolism by mutating T. tepidiphila cultures using UV radiation to knock out said genes. I will compare the whole genome sequences of mutated colonies that are unable to metabolize PAHs to whole genome shotgun sequence3 to see which genes were knocked out by the mutagenic UV radiation. And, finally, I will characterize the effects of T. tepidiphila previously confirmed to contain genes conferring PAH metabolism on phytoplankton populations in a PAH-polluted marine microcosm in order to determine if these effects are protective, detrimental, or neutral.</p>
<h3 id="proposal--methods">Proposal &amp; Methods</h3>
<p>I will be using T. tepidiphila 1-1B isolated from petroleum-contaminated seawater in a beach simulation tank Kamaishi City, Japan, cultivated using marine broth 2216 (MB) and stored in glycerol at -80˚C. I will use the 1-1B glycerol stock to inoculate cells in 5 mL MB for 48 hours at 43˚C and harvested via centrifugation. To select for growth of PAH metabolizing cells, I will resuspend the pellet in 10 mL ONR7a media supplemented with 1 mg/L each of phenanthrene, pyrene, benzopyrene, and 100 mg/L heat-treated Arabian light crude oil2 and inoculate this medium for 11 days at 43˚C with shaking2. Next, I will be using gas-chromatography mass-spectrometry (GC-MS) with the Q Exactive GC Orbitrap mass spectrometer in order to identify the metabolites present after growth in the presence of various PAHs. Ultimately, this technique will produce a mass spectrum such that we will be able to identify specific metabolites based on their individual spectral fingerprints9,10,11. I will be looking to identify the specific intermediate metabolites produced by PAH degradation such as: dihydroxynaphthalene9, cis-phenanthrene dihydrodiols10, 4,5-dihydroxypyrene, 3,4-dihydroxyphthalate11. The presence of these metabolites in the sample will confirm that the T. tepidiphila isolates are capable of PAH metabolism.
Next, I will identify and characterize the genes that are involved in PAH metabolism via knockout mutations and whole genome sequencing. I will use the 1-1B glycerol stock to inoculate cells in 5 mL MB for 48 hours, or until the OD is between 0.5 and 0.7 OD660, at 43˚C with shaking. After inoculation, I will perform a series of five 1:10 serial dilutions in MB and irradiate the experimental aliquots 45 cm underneath a 30-W germicidal UV light in order to induce random mutations within the genome of T. tepidiphila. I will then plate the irradiated and control aliquots onto Marine Agar plates and incubate at 43˚C for 48 hours in order to select for colonies in which essential genes were not mutated. Next, I will inoculate single colonies from these plates in 5 mL MB at 43˚C for 48 hours and make a glycerol stock of each mutant isolate. I will then inoculate 5 mL of 10 mL ONR7a media supplemented with 1 mg/L each of phenanthrene, pyrene, benzopyrene and 100 mg/L heat-treated Arabian light crude oil2 with a sample from a mutant glycerol stocks for each mutant that was isolated. These samples will be incubated for 11 days at 43˚C with shaking2. I will also inoculate this medium with a sample from the wildtype glycerol stock to act as a positive control and incubate a non-inoculated medium sample to act as a negative control. After 11 days, I will measure the optical density of each sample and compare it to the optical density of the positive control while normalizing for the negative control. I will use the Qiagen DNeasy UltraClean Microbial Kit to extract genomic DNA (gDNA) from the cultures as this kit is able to extract high-quality gDNA from microbial cultures and will yield an accurate product for later sequencing. Next, I will perform whole genome sequencing on each of the mutant isolates displaying decreased PAH metabolic ability after mutagenesis using the Illumina HiSeq System. Once I have sequenced the genomes, I will us the Basic Local Alignment Search Tool (BLAST) to compare each mutant genome with the T. tepidiphila whole genome shot gun sequence3; BLAST will find regions that differ between the wildtype the mutant genome and calculate the statistical significance of the comparison.
Next, I will obtain P. marinus (CCMP2937) and M. pusilla (CCMP1545) to carry out the microcosm study. P marinus will be inoculated in batches of 5 L Pro99 medium and M. pusilla will be inoculated in batches of 5 L K medium18. The cultures will be grown in a 12:12 light: dark photoperiod, and, during the exponential growth phase, 250 mL of the respective cultures will be added to a 5 L flask containing 3 mL of autoclaved, artificial sea water (ASW). The concentration of phytoplankton will be determined by optical density after a 48-hour incubation period with shaking to ensure phytoplankton growth is optimal under these conditions. Next, to confirm that T. tepidiphila and the respective phytoplankton can co-exist in this microcosm, 250 mL of the mutated T. tepidiphila cultivated in MB and 250 mL of freshly cultivated phytoplankton will be added to the flask containing ASW. To quantify the amount of P. marinus, T. tepidiphila, and M. pusilla in each microcosm, I will perform RT-qPCR using 16s rRNA specific primers for each of the three strains. After quantifying the bacterium and phytoplankton, I will add 250 mL ONR7a media supplemented with 1 mg/L each of phenanthrene, pyrene, benzopyrene, and 100 mg/L heat-treated Arabian light crude oil2 and inoculate this medium for 11 days at 43˚C with shaking2. After the incubation period, I will repeat RT-qPCR with 16s rRNA primers and compare the concentrations of P. marinus, M. pusilla, and T. tepidiphila before and after the introduction of PAHs.</p>
<h3 id="intellectual-merit">Intellectual Merit</h3>
<p>By identifying the genetic factors conferring PAH metabolism in T. tepidiphila, we can identify specific genetic variations that enable T. tepidiphila to survive in PAH polluted environments, while other vital organisms such as phytoplankton lack this genetic PAH resistance, and, we can identify functional genes that can metabolize these pollutants. Through the identification of the loci and sequences conferring PAH resistance in T. tepidiphila, we can form a framework for understanding the genetic advantages that allow certain organisms to survive in increasingly polluted ecosystems; this may allow a basis for new techniques for the removal of pollutants and toxins from marine and aquatic ecosystems. Through the microcosm study, we can predict how increased PAH pollution will affect phytoplankton populations, as well as determine if co-existence of T. tepidiphila with different groups of phytoplankton helps to stabilize the primary producer population while also facilitating the bioremediation of PAHs. Understanding the population dynamics between phytoplankton and T. tepidiphila while in the presence of PAHs will enable the creation of a predictive model of the primary producer’s viability in the face of increased PAH concentrations.</p>
<h3 id="broader-impact">Broader Impact</h3>
<p>In addition to the potential application the results of this study may have on stabilizing the population of primary producers and removing toxins from marine ecosystems, I believe that my unique experience with scientific research and my journey through academia would make me a valuable research mentor for undergraduates and others seeking advice in the broad field of biological sciences. At 17-years-old I was diagnosed with Systemic Lupus Erythematosus, and, the increasing disease activity as well as the treatments that I underwent while entering my undergraduate studies, and continued to endure throughout, gives me a deeper insight into the specific and unpredictable struggles faced by students with chronic illnesses. With an invisible disability and life-long illness, the competitive and fast paced world of academic research can feel inaccessible, especially when compounded with the lack of representative leaders in the STEM fields who live with, or speak about, chronic illness; through my continued studies with this research proposal and obtaining my Ph.D. in the field of ecology, I can mentor students through the academic process and help them foster and cultivate their research interests in a more accessible way through recognizing where the academic system creates barriers for aspiring researchers living with disabilities and helping to dismantle them.</p>
<h3 id="refernces">Refernces</h3>
<p>1.) Fahnestiel, et al., Elsevier 2010; 2. Kodama, Stiknowati, et al. Int. Jorn. of Sys. and Evo. Micro., 2008; 3. Lai, Shao, The Third Inst. of Ocea., 2014; 4. Wang, Fingas, et al., Jorn. Of Chrom., 1998; 5. Kim, Kwon, Hand. of Hyd. &amp; Lip. Micro. 2010; 6. Karacik, B, et al., Enviro. Int. 2009; 7. Todd, Peter A., et al., Spr-Sci 2010; 8. González, Figueiras, et al., Est. Coast. &amp; Sh. Sci. 2009; 9. Annweiler, E., et al., App. &amp; Env. Micro., 2000; 10. Gao, Shumei, et al., Int. Biodet. &amp; Biodeg. 2013; 11. Kim, Seong-Jae, et al., 2007; 13. Su, Xiaoyang, et al., 2017; 14. Lin, Kevin, and Alice Wang, 2001; 15. Golden, et al., 2004; Cerezo, et al., 2014; 16. Shivaji, et al., 2015; 17. Kruk, et al., 2002;18Echeveste, et al., 2010.</p>

        </div>
        
        <div class="my-4">
    
    <a href="https://ericalittman.com/tags/marketing/" class="inline-block bg-tertiary-bg text-sm rounded px-3 py-1 my-1 mr-2 hover:text-eureka">#marketing</a>
    
    <a href="https://ericalittman.com/tags/social-media/" class="inline-block bg-tertiary-bg text-sm rounded px-3 py-1 my-1 mr-2 hover:text-eureka">#social media</a>
    
    <a href="https://ericalittman.com/tags/portfolio/" class="inline-block bg-tertiary-bg text-sm rounded px-3 py-1 my-1 mr-2 hover:text-eureka">#portfolio</a>
    
</div>
        
        
        
        
        
        
        
<div class="flex flex-col md:flex-row md:justify-between -mx-2 mt-4 px-2 pt-4 border-t">
    <div>
        
        <span class="block font-bold">Previous</span>
        <a href="https://ericalittman.com/posts/untitled_project/" class="block">Prologue to untitled novel project</a>
        
    </div>
    <div class="md:text-right mt-4 md:mt-0">
        
    </div>
</div>

        
    </div>
    
    <div class="col-span-2">
        
        
        <div class="sticky top-16 z-10 hidden lg:block px-6 py-4  bg-primary-bg ">
    <span class="text-lg font-semibold">On This Page</span>
</div>
<div class="sticky-toc hidden lg:block px-6 pb-6 ">
    <nav id="TableOfContents">
  <ul>
    <li><a href="#background">Background</a></li>
    <li><a href="#genetic-factors-that-confer-pah-metabolism-in-thalassospira-tepidiphila">Genetic Factors that Confer PAH Metabolism in Thalassospira tepidiphila</a>
      <ul>
        <li><a href="#objectives">Objectives</a></li>
        <li><a href="#proposal--methods">Proposal &amp; Methods</a></li>
        <li><a href="#intellectual-merit">Intellectual Merit</a></li>
        <li><a href="#broader-impact">Broader Impact</a></li>
        <li><a href="#refernces">Refernces</a></li>
      </ul>
    </li>
  </ul>
</nav>
</div>
<script>
    window.addEventListener('DOMContentLoaded', () => {
        enableStickyToc();
    });
</script>
        
    </div>
    

    
    
</div>
<script>
    document.addEventListener('DOMContentLoaded', ()=>{
        hljs.initHighlightingOnLoad();
    })
</script>

      </div>
    </div>
    
  </main>
  <footer class="pl-scrollbar">
    <div class="w-full max-w-screen-xl mx-auto"><div class="text-center p-6 pin-b">
    <p class="text-sm text-tertiary-text"> </p>
</div></div>
  </footer>
</body>

</html>